### PR TITLE
[RGB_MATRIX] Changed LED positions for Massdrop CTRL and DZ60RGB

### PR DESCRIPTION
**Pull request by MatthewRobo**
## Description
Changed positions of the Massdrop (Drop?) CTRL and the DZ60RGB to more accurately reflect their physical LED positions.
Also changed modifiers to highlight the keys that would come in a keyset's modifier kit.
This means ESC, F5-F8, the nav cluster, non-alphanumeric keys, and more.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
@moyi4681 and @patrickmt, any issues with this?

**Comment by moyi4681** [comment]
LGTM

File: keyboards/massdrop/ctrl/config_led.c [hunk 1]
```diff
@@ -6,132 +6,132 @@
...
-  { { 5|(2<<4) }, { 38, 59 }, 4 },
-  { { 5|(3<<4) }, { 83, 59 }, 4 },
-  { { 5|(4<<4) }, { 129, 59 }, 4 },
-  { { 5|(5<<4) }, { 144, 59 }, 4 },
-  { { 5|(6<<4) }, { 159, 59 }, 4 },
-  { { 5|(7<<4) }, { 174, 59 }, 4 },
-  { { 10|(4<<4) }, { 193, 59 }, 4 },
-  { { 10|(5<<4) }, { 205, 59 }, 4 },
-  { { 10|(6<<4) }, { 217, 59 }, 4 },
+  // R0
```
[the remark this answers is not in the document]
Comment (MatthewRobo) [reply keyboards/massdrop/ctrl/config_led.c]: whoops, i thought i added those back but i guess that was only for the dz60rgb,
gimme a sec

[the remark this answers is not in the document]
Comment (MatthewRobo) [reply keyboards/massdrop/ctrl/config_led.c]: Hold up, I'm pretty sure my keycode comments are still there.
Are you requesting that I change it back to the previous single-line keycode comments?

**Comment by drashna** [comment]
@MatthewRobo Some merge conflicts are present now. 

If you could fix them?

**Comment by patrickmt** [comment]
What's the reasoning behind changing the corner LEDs to NOT reflect their actual physical positioning? The original positions are derived from the PCB layout coordinates.

**Comment by drashna** [comment]
@MatthewRobo It looks like you have some merge conflicts due to the new structure. 

@patrickmt I think that he's trying to change them to be more "accurate" in appearance, rather than physical location.

**Comment by MatthewRobo** [comment]
> @MatthewRobo It looks like you have some merge conflicts due to the new structure.

I think I fixed it, though I might have accidentally added some garbage in my keymaps with and I don't know how to remove it at this point because I suck at git.

**Comment by drashna** [comment]
@patrickmt were you okay with this change, or...?

**Comment by patrickmt** [comment]
Yeah the changes are fine

**Comment by drashna** [comment]
Thanks!

**Pull request merged**